### ISSUE TITLE
Release with fixes for exponential value bug

### ISSUE THREAD

**Issue by sutoh** [geomet/geomet#81]
I would like you to release a new tag with this change.
https://github.com/geomet/geomet/commit/30cc048cb51c4ac1b85374d31b6bcd28c0351623#diff-d876c01bf5eb778d44feb3d3ef88585c74dde2e79364dbcda7854c4c87a52052R200-R203

The error I am facing now occurs with the following command.

```
wkt.dumps({'type': 'Point', 'coordinates': [4e-05, 51.530281]})
```

This is fixed in this commit, but since the release tag is `0.3.0`, it cannot be specified in the requrements.txt file.

**Comment by larsbutler** [comment]
Hi @sutoh , thanks for bringing this up. I'll get to work on a release. Unfortunately it looks like the release process is broken, and some of the packages required for release have dropped support for old Python versions. It looks like we'll need to drop support for EOL'ed versions of python (3.6 and older) in order to get this released. Bear with us.

**Comment by larsbutler** [comment]
Hi @sutoh, this is fixed now. See https://github.com/geomet/geomet/releases/tag/1.0.0 and https://pypi.org/project/geomet/1.0.0/.

Fixed by #87.
